lib/chef/knife/bootstrap/bootstrap_options.rb

```diff
@@ -90,6 +90,13 @@ def self.included(includer)
                                   It specifies the frequency (in minutes) at which the chef-service runs.
                                   Pass 0 if you don't want the chef-service to be installed on the target machine."
 
+            option :daemon,
+              :long => "--daemon DAEMON",
+              :description => "Optional, Provide this option when --bootstrap-protocol is set to 'cloud-api'. Works only for Windows node.
+                                Options: 'auto' or 'service'.
+                                auto - Provide when you don't want to run the chef-client as service.
+                                service - Provide when you want to run chef-client as service"
```

Comment:
service - Configures the chef-client to run automatically in the background as a service

or "as a service or via cron" if this affects linux?